domains/transfer/Transfer.API/API.IntegrationTests/SwaggerTests.GetSwaggerDoc_AppStarted_NoChangesAccordingToSnapshot.verified.txt

```diff
@@ -101,53 +101,20 @@
         }
       }
     },
-    "/api/connection-invitations": {
```

Comment:
It is a breaking change to remove endpoints related to "Connections". Do you have a strategy/agreement with Alexander on how not to break the frontend?

Reply:
Big bang :D 
Will coordinate with him